src/storage/src/storage_state.rs

```diff
@@ -207,6 +284,19 @@ impl<'w, A: Allocate> Worker<'w, A> {
                         ))),
                     );
 
+                    self.storage_state.sink_handles.insert(
```

Comment:
should we only do this on one worker?

Reply:
oh wait does the sink only run on 1 worker?

Reply:
I've double-checked the sink, and it does correctly drop to the empty antichain for workers that aren't actively writing to Kafka. So I think this behaviour is fine... we'll have a few tasks that aren't doing much after dropping to the empty antichain, but tasks are pretty cheap. (And I like having as few special-cases as possible.)